docs/README.md

```diff
@@ -41,117 +41,74 @@ In the product theme template, add the `condition-layout.{context}` block, repla
-| `areAllVariationsSelected` | `value` | Whether all product variations available on the page were selected by the user (`true`) or not (`false`). | 
-
-:information_source: *Since the Condition Layout can only be used with product contexts, only the subjects listed above are needed for the proper functioning of the `condition` block. Remember to choose the subject's value according to the value passed to the `object` prop*.
+| Subject                    | Description            | Arguments      |
+| -------------------------- | ---------------------- | -------------- |
+| `productId`                | Product's IDs currently displayed on the UI.  | `{ id: string }` |
+| `categoryId`               | Category's IDs currently displayed on the UI. | `{ id: string }` |
+| `brandId`                  | Brand's IDs currently displayed on the UI.    | `{ id: string }` |
+| `selectedItemId`           | ID of the item currently selected by the user.   | `{ id: string }` |
+| `productClusters`          | List of product clusters currently displayed on the UI.   | `{ id: string }` |
+| `categoryTree`             | List of categories currentluy displayed on the UI.   | `{ id: string }`  |
+| `specificationProperties`  | List of product specifications currently displayed on the UI. | `{ name: string, value: string }`. Notice: `value` is an optional prop. If omitted, only the specification name (`name`) will be checked. |
+| `areAllVariationsSelected` | Whether all product variations currently available on the UI were selected by the user (`true`) or not (`false`). | No arguments are expected. |
 
 ## Modus Operandi
 
-In practice, the Condition Layout does not render a block on its own. **The app provides 3 logic blocks**, meaning blocks that lay out the reasoning behind rendering other Store Framework blocks.
-
-The `condition.{context}` block is the one that does your store's actual Layout logic and, using the `conditions` and `match` props to set the conditions that blocks must meet to be rendered or not.
+The `condition-layout.product` uses the `conditions` and `match` props to set the conditions that blocks must meet to be rendered or not.
 
-The `conditions` prop object has 3 other props, namely `subject`, `verb` and `object`, that together define the condition that must be met and how it is going to be validated: the `object` prop from `conditions` compares its value with the values fetched by the subject passed to the `subject` prop. The criteria used for this comparison stems from the value passed in the `verb`. The result being to define whether the condition put forth by the `condition` block and its props is acuatlly valid or not. 
+The `conditions` object has two props, namely `subject` and `arguments`, that together define one condition that must be met. Each `subject` has an underlying validation method executed whenever its `subject` is used. The `arguments` object can be considered arguments being sent to the underlying method.
```

Comment:
Instead of having a magical grammar that allows us to compare if a `subject` `verb`(is,is-not, etc) an `object`, we switched to a more specific way of doing conditions. 

Given this condition:

```
{
  'subject': 'specificationProperties'
  'arguments': {
    'name': 'Potato'
  }
}

```

If we converted it into code, we would have:

```
specificationProperties({ name: 'potato' })
```

The point is that we don't have a condition grammar anymore. Instead, we have ways to call methods that will validate these conditions.